fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/tree/lowcardinality/DecodeCollector.java

```diff
@@ -173,6 +253,9 @@ private void initContext(DecodeContext context) {
             if (context.allStringColumns.contains(cid)) {
                 continue;
             }
+            if (disableRewriteStringColumns.contains(cid)) {
```

Comment:
duplciate with line:226?

Reply:
226 is check each scanColumns
256 is check columns generated by project expression